runtime/common/src/lib.rs

```diff
@@ -14,6 +15,25 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with Zeitgeist. If not, see <https://www.gnu.org/licenses/>.
+//
+// This file incorporates work covered by the following copyright and
+// permission notice:
+//
+//     Copyright (C) 2020-2022 Acala Foundation.
```

Comment:
Acala Foundation :thinking:

Reply:
Yeah, `DealWithFees` is (sort of) taken from [here](https://github.com/paritytech/substrate/blob/e77cbe39c4d1bfd978bb03c686fc9f60d86a3d06/frame/transaction-payment/src/lib.rs#L966-L978), a s Chris pointed out.

Reply:
I also included Parity in the copyright because the test for `DealWithFees` was taken from their runtime implementation.